TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -343,10 +334,13 @@ private void TryPopModal()
                 }
             }
 
-            if (blurEffect != null && UIView.ModalInputCount() == 0) {
-                blurEffect.eventPositionChanged -= OnBlurEffectPositionChange;
-                blurEffect.eventZOrderChanged -= OnBlurEffectZOrderChange;
```

Comment:
And this... :)

Reply:
Let me double check if i pulled in a wrong version or fat fingered somewhere, or something

Reply:
`JB Rider` don't like delegate `-=` so maybe removed that automatically

Reply:
Wrong version, reapplied the changes

Reply:
I had master from 10.21, not the fixed 10.21.1, very sorry
It was wrong idea to touch that file anyway :)

Reply:
No problem :)